packages/app/cypress/e2e/cypress-in-cypress-component.cy.ts

```diff
@@ -62,17 +62,17 @@ describe('Cypress In Cypress CT', { viewportWidth: 1500, defaultCommandTimeout:
       cy.get('[data-model-state="passed"]').should('contain', 'renders the test component')
 
       // go to Settings page and back to spec runner
-      cy.contains('a', 'Settings').click()
+      cy.findByTestId('nav-row-settings').click()
```

Comment:
I like how you updated all these to use consistent commands/selectors. 🥳 

Have you considered writing a custom cypress command to handle these 'navigate to route' flows that are so well defined? Something like:

```js
// packages/app/cypress/e2e/support/e2eSupport.ts

Cypress.Commands.add('navigateAppTo', (route: 'specs' | 'runs' | 'settings') => {
  return cy.findByTestId(`nav-row-${route}`).click()
})


// usage
cy.navigateAppTo('runs')
```

Might be a little more readable?

Reply:
Mark made a good point that I mistakenly turned this anchor tag into a button 😅 it's been reverted back to anchor tag so no diffs here 🎉